src/components/organisms/Dashboard.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import ReactGridLayout from 'react-grid-layout';
+import { ParentSize } from '@visx/responsive';
+
+import { Pane, MinimizeIcon, MaximizeIcon, IconButton } from 'evergreen-ui';
+
+import DashboardItem from '../molecules/DashboardItem';
+import { DashboardItemSize, DashboardItemInfo, DashboardProps } from '../../dataTypes/Dashboard/DashboardTypes';
+
+import 'react-grid-layout/css/styles.css';
+import 'react-resizable/css/styles.css';
+
+type Layout = {
+    i: string;
```

Comment:
It seems like there are more conversions from int to string than reverse, so you could type this as an int.

And with that replace `Layout` with `Layout = Record<'i' | 'x' | 'y' | 'w' | 'h', number>;`.